docs/src/persistence.md

```diff
@@ -1,14 +1,10 @@
 # Reading and writing Graphs
 
-Graphs may be written to I/O streams and files using the `savegraph` function and read with the `loadgraph` function. The default graph format is a proprietary compressed *LightGraphs.jl* format `lg`. Other formats are available via the *GraphPersistence* package.
+## Saving using *LightGraphs.jl* `lg` format.
 
-```@autodocs
-Modules = [LightGraphs]
-Pages   = [ "persistence/common.jl"]
-Private = false
-```
+Graphs may be written to I/O streams and files using the `savegraph` function and read with the `loadgraph` function. The default graph format is a bespoke compressed *LightGraphs.jl* format `lg`.
 
-## Examples
+### Example
 
 ```julia
 save(STDOUT, g)
```

Comment:
@sbromberger It occurs to me this example doesn't quite jive with the text. What is the preferred way to save / load now?

Reply:
`savegraph` / `savegraphs`